webpack.config.js

```diff
@@ -1,102 +1,96 @@
-/* eslint-env node */
-const path = require("path");
-const webpack = require("webpack");
-const ExtractTextPlugin = require("extract-text-webpack-plugin");
-const extractStyles = new ExtractTextPlugin("main.css");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const externalAssets = require("./lib/javascripts/external_assets");
+const path = require('path')
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const TranslationsPlugin = require('./webpack/translations-plugin')
+const devDependencies = require('./package.json').devDependencies
+
+// this function reads Zendesk Garden npm dependencies from package.json and
+// creates a jsDelivr url
+const zendeskGardenJsDelivrUrl = (function () {
+  const pkg = Object.keys(devDependencies).filter(item => item.includes('@zendeskgarden/css'))
+
+  return pkg.length && pkg.reduce(
+    (url, pkg) => {
```

Comment:
Do we put the function arguments on a new line now? Does that mean every function code will have 4 spaces indent from now on?

Reply:
I don't think there's any convention on this, all depends on the readability in the context.
This statement is chaining a few expressions, also use `reduce` which is hard to read with a inline callback definition. That's the main reason to format it on different lines.
I'm happy to extract the callback in a separate function if it helps.